src/main/resources/com/google/api/codegen/java/settings.snip

```diff
@@ -102,6 +110,30 @@
     }
 
   @end
+
+  /**
+   * Returns a builder for the default credentials for this service.
+   */
+  public static GoogleCredentialsProvider.Builder defaultCredentialsProviderBuilder() {
+    return GoogleCredentialsProvider.newBuilder()
+        @if xsettingsClass.hasDefaultServiceScopes
+            .setScopesToApply(DEFAULT_SERVICE_SCOPES)
+        @end
+        ;
```

Comment:
I'm inclined to not fight with snippet formatting since GoogleJavaFormat fixes it anyway.